static/app/components/replays/breadcrumbs/utils.tsx

```diff
@@ -3,17 +3,23 @@ import {IconWarning} from 'sentry/icons';
 import {t} from 'sentry/locale';
 import {BreadcrumbType, Crumb} from 'sentry/types/breadcrumbs';
 
+// Replay SDK can send `data` that does not conform to our issue/event breadcrumbs
+type MaybeCrumbData = null | Record<string, unknown>;
+
 /**
  * Generate breadcrumb descriptions based on type
  */
 export function getDescription(crumb: Crumb) {
-  if (typeof crumb.data === 'object' && crumb.data !== null && 'action' in crumb.data) {
-    switch (crumb.data.action) {
+  const crumbData: MaybeCrumbData = crumb.data as MaybeCrumbData;
+
+  if (typeof crumbData === 'object' && crumbData !== null && 'action' in crumbData) {
```

Comment:
`typeof crumbData === 'object' && crumbData !== null`

thanks javascript for making null an object 🤯 💥  🔫

Reply:
This can be written with slighly less bytes as:

```js
if( crumbData && typeof crumbData === 'object' && 'action' in crumbData)
```